src/styles/index.scss

```diff
@@ -1,3 +1,243 @@
+$color-link: #00acdc;
+$size-box: 200px;
+$pad-box: 16px;
+$size-img: calc($size-box - $size-box / 5);
+$size-40: 40px;
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+  height: #{$size-40};
+  width: #{$size-40};
+
+  &__image {
+    height: 100%;
+    width: 100%;
+  }
+}
+
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    list-style: none;
+  }
+
+  &__item {
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 60px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    white-space: nowrap;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #060b35;
+    transition: color 0.3s;
+  }
+
+  &__link.is-active::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 4px;
+    border-radius: 2px;
+    background-color: #{$color-link};
+  }
+
+  &__link.is-active,
+  &__link:hover {
+    color: #{$color-link};
+  }
+}
+
+ul {
+  display: block;
+  list-style-type: disc;
+  margin-block: 1em;
+  margin-inline: 0;
+  padding-inline-start: #{$size-40};
+}
+
+li {
+  display: list-item;
+  text-align: -webkit-match-parent;
+}
+
+.catalog {
+  margin: #{$size-40} 50px;
+  padding: 10px 20px;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  gap: 7.2px;
+  margin-left: 1.3px;
```

Comment:
just use the correct margin (4px)

Note that the margin should be only between stars (like gap, so basically you can use gap, but make sure you set correct value and tests pass), so no left margin for first star or vice versa - no right margin for the last star